src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Please include a header comment to indicate what this interface/class/method does?